robottelo/ui/contentenv.py

```diff
@@ -11,53 +11,42 @@ class ContentEnvironment(Base):
     def create(self, name, description=None, prior=None):
         """Creates new life cycle environment."""
         if prior:
-            strategy, value = locators["content_env.env_link"]
-            element = self.wait_until_element((strategy, value % prior))
-            if element:
-                element.click()
+            strategy, value = locators['content_env.env_link']
+            self.click((strategy, value % prior))
         else:
-            self.wait_for_ajax()
-            self.find_element(locators["content_env.new"]).click()
-        if self.wait_until_element(common_locators["name"]) is None:
+            self.click(locators['content_env.new'])
+        if self.wait_until_element(common_locators['name']) is None:
             raise UINoSuchElementError(
                 'Could not create new environment {0}'.format(name))
-        self.text_field_update(common_locators["name"], name)
+        self.text_field_update(common_locators['name'], name)
         if description:
-            self.text_field_update(common_locators["description"],
-                                   description)
-        self.wait_until_element(common_locators["create"]).click()
+            self.text_field_update(
+                common_locators['description'], description)
+        self.click(common_locators['create'])
 
     def delete(self, name):
         """Deletes an existing environment."""
-
-        strategy = locators["content_env.select_name"][0]
-        value = locators["content_env.select_name"][1]
-        element = self.wait_until_element((strategy, value % name))
-        if element is None:
-            raise UINoSuchElementError(
-                'Could not find the selected environment {0}'.format(name))
-        element.click()
-        self.wait_until_element(locators["content_env.remove"]).click()
+        strategy = locators['content_env.select_name'][0]
+        value = locators['content_env.select_name'][1]
```

Comment:
This can be converted to a single line, by 
strategy, value = locators['content_env.select_name']

Please fix other instances too.